HappyAnding/HappyAnding/Views/Components/ListShortcutView.swift

```diff
@@ -7,9 +7,59 @@
 
 import SwiftUI
 
+/// 카테고리에서 해당 뷰를 호출할 때에는 categoryName에 카테고리 이름을 넣어주세요
 struct ListShortcutView: View {
+    
+    @ObservedObject var shortcutData = fetchData()
+    
+    var categoryName: String?
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        
+        List {
+            
+            header
+                .listRowBackground(Color.Background)
+                .listRowSeparator(.hidden)
+                .listRowInsets(EdgeInsets())
+            
+            ForEach(0..<shortcutData.data.count, id: \.self) { index in
+                ShortcutCell(color: self.shortcutData.data[index].color,
+                             sfSymbol: self.shortcutData.data[index].sfSymbol,
+                             name: self.shortcutData.data[index].name,
+                             description: self.shortcutData.data[index].description,
+                             numberOfDownload: self.shortcutData.data[index].numberOfDownload,
+                             downloadLink: self.shortcutData.data[index].downloadLink)
+                .listRowInsets(EdgeInsets())
+                .listRowSeparator(.hidden)
+                .onAppear {
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
+                        if index == shortcutData.data.count - 1 && index < 99 {
+                            self.shortcutData.updateData()
+                        }
+                    }
+                }
+                
+            }
+        }
+        .listRowBackground(Color.Background)
+        .listStyle(.plain)
+        .background(Color.Background.ignoresSafeArea(.all, edges: .all))
+        .scrollContentBackground(.hidden)
+        .navigationBarTitle("다운로드 순위")
```

Comment:
```suggestion
        .scrollContentBackground(.hidden)
        .navigationBarTitle(categoryName)
```
뷰 재사용을 위해서 여기서도 categoryName을 전달하는 것은 어떨까요?

Reply:
오 감사합니다! 사랑받는 앱도 같이 보여주기위해서는 변수가 또 필요하겠네요